actor-tests/src/test/scala/org/apache/pekko/util/ByteStringSpec.scala

```diff
@@ -717,6 +717,82 @@ class ByteStringSpec extends AnyWordSpec with Matchers with Checkers {
       compact.indexOf('g', 5) should ===(5)
       compact.indexOf('g', 6) should ===(-1)
     }
+    "indexOf (specialized)" in {
```

Comment:
add a blank line

Reply:
I'm a bit confused. Where should I add the blank line? After? Before?

This was copied from the tests above, which don't seem to have any blank lines 😕

Reply:
I'm ok with you keeping the existing style. Blanks lines are not needed if the existing style is like that.